docs/tutorial/helm-install.md

```diff
@@ -14,21 +16,8 @@ Or, some of Helm v3 Latest Release on Aliyun OSS:
 If you are using Helm for first time, you may need to init Helm:
 
 ```
-➜  ~ helm init
-Creating ~/.helm
-Creating ~/.helm/repository
-Creating ~/.helm/repository/cache
-Creating ~/.helm/plugins
-Creating ~/.helm/starters
-Creating ~/.helm/cache/archive
-Creating ~/.helm/repository/repositories.yaml
-Adding stable repo with URL: https://kubernetes-charts.storage.googleapis.com
-^C
-
-➜  ~ touch ~/.helm/repository/repositories.yaml
-
+helm init
```

Comment:
why the workaround is not needed now ?

Reply:
The best way would be `helm init --stable-repo-url https://apphub.aliyuncs.com`, will amend it.